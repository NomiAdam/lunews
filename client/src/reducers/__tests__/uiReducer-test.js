/* eslint-disable no-undef */
import {
    unsetNotification, setNotification, setStatePosition, setNavigation, setSearchValue, setOfflineDialog,
} from '../uiReducer';

const mockPayload = 'mock';

describe('uiReducer', () => {
    describe('uiReducerActionCreators', () => {
        it('should match the snapshot for setNotification', () => {
            expect(setNotification(mockPayload)).toMatchSnapshot();
        });
        it('should match the snapshot for unsetNotification', () => {
            expect(unsetNotification).toMatchSnapshot();
        });
        it('should match the snapshot for setStatePosition', () => {
            expect(setStatePosition(mockPayload)).toMatchSnapshot();
        });
        it('should match the snapshot for setStatePosition', () => {
            expect(setNavigation(mockPayload)).toMatchSnapshot();
        });
        it('should match the snapshot for setSearchValue', () => {
            expect(setSearchValue(mockPayload)).toMatchSnapshot();
        });
        it('should match the snapshot for setOfflineDialog', () => {
            expect(setOfflineDialog(mockPayload)).toMatchSnapshot();
        });
    });
});
