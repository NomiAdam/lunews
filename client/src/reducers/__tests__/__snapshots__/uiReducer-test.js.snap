// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uiReducer uiReducerActionCreators should match the snapshot for setNotification 1`] = `
Object {
  "payload": "mock",
  "type": "ui/SET_NOTIFICATION",
}
`;

exports[`uiReducer uiReducerActionCreators should match the snapshot for setOfflineDialog 1`] = `
Object {
  "payload": "mock",
  "type": "ui/SET_OFFLINE_DIALOG",
}
`;

exports[`uiReducer uiReducerActionCreators should match the snapshot for setSearchValue 1`] = `
Object {
  "payload": "mock",
  "type": "ui/SET_SEARCH_VALUE",
}
`;

exports[`uiReducer uiReducerActionCreators should match the snapshot for setStatePosition 1`] = `
Object {
  "payload": "mock",
  "type": "ui/SET_STATE_POSITION",
}
`;

exports[`uiReducer uiReducerActionCreators should match the snapshot for setStatePosition 2`] = `
Object {
  "payload": "mock",
  "type": "ui/SET_NAVIGATION",
}
`;

exports[`uiReducer uiReducerActionCreators should match the snapshot for unsetNotification 1`] = `
Object {
  "type": "ui/CLEAR_NOTIFICATION",
}
`;
